ui/src/literature/components/LiteratureClaimButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ClaimingDisabledButton from '../../authors/components/ClaimingDisabledButton';
+import AssignNoProfileAction from '../../authors/components/AssignNoProfileAction';
+import AssignLiteratureItemContainer from '../containers/AssignLiteratureItemContainer';
+import NoAuthorsClaimingButton from './NoAuthorsClaimingButton';
+
+const LiteratureClaimButton = ({
+  loggedIn,
+  hasAuthorProfile,
+  authors,
+  controlNumber,
+}: {
+  loggedIn: boolean;
+  hasAuthorProfile: boolean;
+  authors: [];
+  controlNumber: string;
+}) => (
+    <>
+      {!loggedIn && <ClaimingDisabledButton />}
+      {!hasAuthorProfile && <AssignNoProfileAction />}
+      {!authors && <NoAuthorsClaimingButton />}
```

Comment:
!authors is not correct, what exactly you would like to check? Perhaps if `authors.length === 0`?